runbotics/runbotics-orchestrator-ui/src/src-app/views/process/AddProcess.tsx

```diff
@@ -61,9 +61,18 @@ const AddProcessDialog: FC<AddProcessDialogProps> = ({
     const { translate } = useTranslations();
     const dispatch = useDispatch();
     const { enqueueSnackbar } = useSnackbar();
+    const isFieldEmpty = name ? name.trim() === '' : false;
```

Comment:
can be done in a way `Boolean(name && name.trim().length === 0)`

boolean is optional, but if we don't want undefined you can add it (undefined is falsy anyway)